stock_release_channel_auto_release/data/queue_job_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record model="queue.job.channel" id="stock_release_channel">
+        <field name="name">stock_release_channel</field>
+        <field name="parent_id" ref="queue_job.channel_root" />
+    </record>
+
+    <record id="job_function_stock_picking_auto_release" model="queue.job.function">
+        <field name="model_id" ref="stock.model_stock_picking" />
+        <field name="method">auto_release</field>
+        <field name="channel_id" ref="stock_release_channel" />
+    </record>
```

Comment:
Looks like a copy/paste that was not adapted to this module method (cc @lmignon )

```suggestion
     <record model="queue.job.channel" id="stock_release_channel_auto_release">
         <field name="name">stock_release_channel_auto_release</field>
         <field name="parent_id" ref="queue_job.channel_root" />
     </record>
 
     <record id="job_function_stock_picking_auto_release" model="queue.job.function">
         <field name="model_id" ref="stock.model_stock_picking" />
         <field name="method">auto_release_available_to_promise</field>
         <field name="channel_id" ref="stock_release_channel_auto_release" />
     </record>
```